concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+Williamson-1973-118-8	8	bean	haricot	Bohne	832	BEAN
+Williamson-1973-118-9	9	beg (=ask for something)	demander (qqc à qqn)	bitten um, erbitten, betteln	1828	ASK (REQUEST)
+Williamson-1973-118-10	10	belly	ventre	Bauch, Unterleib	1251	BELLY
+Williamson-1973-118-11	11	bite	mordre	beissen	1403	BITE
+Williamson-1973-118-12	12	buffalo	buffle	Büffel	934	BUFFALO
+Williamson-1973-118-13	13	build	bâtir	bauen	1840	BUILD
+Williamson-1973-118-14	14	calabash	calebasse	Kalebasse	411	GOURD
+Williamson-1973-118-15	15	carve	sculpter	schnitzen	228	CARVE
+Williamson-1973-118-16	16	charcoal (coal)	charbon	Holzkohle (Kohle)	2659	CHARCOAL OR COAL
+Williamson-1973-118-17	17	child	enfant	Kind	2099	CHILD
+Williamson-1973-118-18	18	come out	sortir	herauskommen	1107	GO OUT
+Williamson-1973-118-19	19	cool, be(come)	se refraîchir	kühl sein, kühl werden		
+Williamson-1973-118-20	20	corpse	cadavre	Leichnam, Leiche, der Tote	767	CORPSE
+Williamson-1973-118-21	21	count: draw a line:: reckon	compter: tirer un ligne:: calculer	zählen: eine Linie ziehen: rechnen		
+Williamson-1973-118-22	22	cow	vache	Kuh	1007	COW
+Williamson-1973-118-23	23	crab	crabe	Krabbe	2348	CRAB
+Williamson-1973-118-24	24	crocodile	crocodile	Krokodil	1857	CROCODILE
+Williamson-1973-118-25	25	dance v. and n.	danser: danse	tanzen: Tanz		
+Williamson-1973-118-26	26	depth (of water)	profondeur (de l’eau)	Tiefe (vom Wasser)	2534	DEPTH
+Williamson-1973-118-27	27	die: death	mourir: mort	sterben: Tod	3186	BE DEAD OR DEATH OR DIE
+Williamson-1973-118-28	28	divide	diviser	teilen, verteilen, einteilen	1119	DIVIDE
+Williamson-1973-118-29	29	draw (water)	puiser (de l'eau)	(Wasser) schöpfen, (Wasser) holen	1911	DRAW (WATER)
+Williamson-1973-118-30	30	dry, become	se dessécher	trocken werden; trocknen	1786	DRY UP
+Williamson-1973-118-31	31	ear	oreille	Ohr	1247	EAR
+Williamson-1973-118-32	32	eat	manger	essen; fressen (animals)	1336	EAT
+Williamson-1973-118-33	33	egg	oeuf	Ei; Hühnerei (hen’s egg, chicken’s egg)	744	EGG
+Williamson-1973-118-34	34	elephant	éléphant	Elefant	1290	ELEPHANT
+Williamson-1973-118-35	35	extinguished, be	s’éteindre	erlöschen (vom Licht); ausgelöscht sein	2209	BECOME EXTINGUISHED
+Williamson-1973-118-36	36	eye	oeil	Auge	1248	EYE
+Williamson-1973-118-37	37	fall	tomber	fallen, stürzen, abstürzen, niederfallen, herunterfallen	1280	FALL
+Williamson-1973-118-38	38	fat: oil	graisse: huile	Fett: Öl (Palmöl)	2551	ORGANIC FAT OR OIL
+Williamson-1973-118-39	39	fear	peur	Furcht, Angst	781	FEAR (FRIGHT)
+Williamson-1973-118-40	40	female	femelle	weiblich	1551	FEMALE
+Williamson-1973-118-41	41	firewood	bois de feu	Brennholz, Feuerholz	10	FIREWOOD
+Williamson-1973-118-42	42	forest	forêt: brousse	Wald, Forst, Urwald:Gebüsch	420	FOREST
+Williamson-1973-118-43	43	fowl (chicken)	volaille	Huhn	265	FOWL
+Williamson-1973-118-44	44	fry: bake in ashes	faire frire: faire cuire	braten: in Asche backen	2146	ROAST OR FRY
+Williamson-1973-118-45	45	give: "dash"	donner: faire cadeau	geben: schenken	1447	GIVE
```

Comment:
Quotation marks are mixed throughout the list. I'd opt for `«` and `»` as is done for other lists where quotation marks are required.